src/utils/results-reporter.ts

```diff
@@ -1,10 +1,18 @@
 import { getOctokit } from "@actions/github";
-import { RunAllTestsResult, RunAllTestsTestRun } from "@alwaysmeticulous/cli";
+import { Project } from "@alwaysmeticulous/api";
+import {
+  ExecuteTestRunResult,
+  RunningTestRunExecution,
+} from "@alwaysmeticulous/sdk-bundles-api";
 import { CodeChangeEvent } from "../types";
 import { updateStatusComment } from "./update-status-comment";
 
 const SHORT_SHA_LENGTH = 7;
 
+export interface EnrichedProject extends Project {
+  isGitHubIntegrationActive?: boolean;
```

Comment:
I guess this will move to the underlying type in future versions of `@alwaysmeticulous/api`?

Reply:
Yep! I'll add it in an SDK PR.